### PR TITLE
Review me pls 

### PR DESCRIPTION
**Pull request by Mazel-Tovr**

### CODE REVIEW

File: TestServer/src/main/java/com/epam/TestServer.java [hunk 1]
```diff
@@ -14,7 +11,7 @@
     private HttpServer httpServer;
 
     public void startServer() throws IOException {
-        httpServer = HttpServer.create(new InetSocketAddress("localhost",1234), 0);
+        httpServer = HttpServer.create(new InetSocketAddress("localhost",0), 0);
```
Comment (LikhanovMaxim) [inline TestServer/src/main/java/com/epam/TestServer.java:14]: formatting

File: TestServer/build.gradle.kts [hunk 2]
```diff
@@ -14,6 +16,24 @@ dependencies {
...
 
 
+val register = tasks.register("startServer", CustomTask::class) {
+    dependsOn("jar")
+    classpath = files("build/libs/TestServer-1.0.0.jar")
+}
+
+
+tasks.register("stopServer"){
+    println("I should stop server but i'm int implemented what a disaster, isn't it ?")
```
Comment (LikhanovMaxim) [inline TestServer/build.gradle.kts:26]: i'm int
what is it?

File: build.gradle.kts [hunk 3]
```diff
@@ -64,22 +66,53 @@ dependencies {
...
+    main = "com.epam.TestKt"
+    println("Hey")
+    dependsOn(tasks["run"])
+}
+
+
+
+tasks["run"].run {
+
+//    val pool = Executors.newFixedThreadPool(5)
```
Comment (LikhanovMaxim) [inline build.gradle.kts]: it should be deleted or add comments

File: src/test/kotlin/unit/service/UserServiceTest.kt [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+//import org.junit.*
+//import org.junit.Test
+//import org.junit.runner.*
+//import org.mockito.*
+//import org.mockito.BDDMockito.*
+//import org.mockito.junit.*
+//import kotlin.test.*
+//
+////@RunWith(value = MockitoJUnitRunner::class)
+//class UserServiceTest {
```
Comment (LikhanovMaxim) [inline src/test/kotlin/unit/service/UserServiceTest.kt:16]: it should be deleted or add comments

**Review by LikhanovMaxim — CHANGES_REQUESTED** [review]

File: TestServer/src/main/java/com/epam/TestServer.java (hunk 1 above)
Comment (Mazel-Tovr) [reply TestServer/src/main/java/com/epam/TestServer.java:14]: fixed in https://github.com/Mazel-Tovr/kotlin-app/pull/6

File: src/test/kotlin/unit/service/UserServiceTest.kt (hunk 4 above)
Comment (Mazel-Tovr) [reply src/test/kotlin/unit/service/UserServiceTest.kt:16]: fixed in https://github.com/Mazel-Tovr/kotlin-app/pull/6

File: TestServer/build.gradle.kts (hunk 2 above)
Comment (Mazel-Tovr) [reply TestServer/build.gradle.kts:26]: fixed in https://github.com/Mazel-Tovr/kotlin-app/pull/6

**Pull request merged**